agent/format-2.0.go

```diff
@@ -60,7 +60,7 @@ type format_2_0Serialization struct {
 	SystemIdentity     string `yaml:"systemidentity,omitempty"`
 	MongoVersion       string `yaml:"mongoversion,omitempty"`
 	MongoMemoryProfile string `yaml:"mongomemoryprofile,omitempty"`
-	MongoSnapChannel   string `yaml:"mongosnapchannel,omitempty"`
+	JujuDBSnapChannel  string `yaml:"juju-db-snap-channel,omitempty"`
```

Comment:
Perhaps can we make this `jujudb-snap-channel`

The other fields don't have hyphens (they should) and reducing the number we use in this new field looks a little tidier IMO